members/serializers.py

```diff
@@ -0,0 +1,26 @@
+from rest_framework import serializers
+
+from members.models import Member
+from members.permissions import has_permission
+from core.serializers import SemesterSerializer
+
+
+class MemberSerializer(serializers.ModelSerializer):
+    semester = SemesterSerializer(read_only=True)
+
+
+
+    class Meta:
+        model = Member
+        fields = ('id', 'name', 'email', 'date_joined', 'semester', 'lifetime', 'honorary')
+
+
+class AddMemberSerializer(serializers.Serializer):
+    '''
+    class Meta:
+        model = Member
+        field = ('id', 'name', 'email', 'lifetime', 'honorary', 'user')
+    '''
+    name = serializers.CharField(max_length=50, help_text='name of user, max 50 letters')
```

Comment:
Kan putte det som en improvement. Jeg ser f.eks. for meg at hvis man får noen form for validering av brukernavn så vil det være en kjappere måte å registrere folk på enn i dag. Da slipper man sannsynligvis både navn og e-post, samt legger til rette for å putte dette inn på studentkort til folk med elektronisk medlemskort.